frontend/containers/project-form-pages/pages/general-information.tsx

```diff
@@ -0,0 +1,275 @@
+            </span>
+            <FieldInfo
+              infoText={formatMessage({
+                defaultMessage:
+                  'Draw on the map or upload a file with the geographical area your project will have an impact on.',
+                id: 'YEYmEz',
+              })}
+            />
+          </Label>
+          {/* Shapefile button - location */}
+          {/* Map - location */}
+          <ErrorMessage id="name" errorText={errors?.location?.message} />
+        </div>
+        <div className="mb-6.5">
+          <p className="mb-2.5 text-gray-600">
+            <FormattedMessage defaultMessage="Project developers" id="0wBg9P" />
+          </p>
+          <div>
+            {/* This fieldset is not a part of the form, it is just to control the visibility of the project developers select. It will be removed from the payload */}
+            <fieldset name="involved_project_developer">
+              <legend className="inline font-sans font-semibold text-sm text-gray-800 mb-4.5">
+                <span className="mr-2.5">
+                  <FormattedMessage
+                    defaultMessage="Are there other Project developers involved in the project?"
+                    id="Xk8qys"
+                  />
+                </span>
+                <FieldInfo
+                  infoText={formatMessage({
+                    defaultMessage: 'Do you have a partnership with someone else?',
+                    id: 'nbqoY2',
+                  })}
+                />
+              </legend>
+              <Label className="block font-normal">
+                <input
+                  id="involved-project-developer"
+                  type="radio"
+                  value={1}
+                  className="mr-2"
+                  {...register('involved_project_developer')}
+                  onChange={() => setShowInvolvedProjectDevelopers(true)}
+                />
+                <FormattedMessage defaultMessage="Yes" id="a5msuh" />
+              </Label>
+              <Label className="block mt-4 font-normal">
+                <input
+                  id="involved-project-developer"
+                  type="radio"
+                  value={0}
+                  className="mr-2"
+                  {...register('involved_project_developer', {
+                    onChange: () => setShowInvolvedProjectDevelopers(false),
+                  })}
+                />
+                <FormattedMessage defaultMessage="No" id="oUWADl" />
+              </Label>
+              <ErrorMessage
+                id="involved-project-developer-error"
+                errorText={(errors?.involved_project_developer as FieldError)?.message}
+              />
+            </fieldset>
+            {/* Project developers selector */}
+            <div
+              className={cx('relative top-[-66px] left-16 w-[360px]', {
+                hidden: !showInvolvedProjectDevelopers,
+              })}
+            >
+              <Combobox
```

Comment:
🟠

You can use the new `<MultiCombobox />` for this now. It was merged very recently. Be careful, the value needs to be an array and you need to use React Hook Forms' `NestedValue` type (check the examples on Storybook).